core/src/Streamly/Internal/FileSystem/File.hs

```diff
@@ -476,34 +482,37 @@ write = writeWith defaultChunkSize
 
 -- | Append a stream of arrays to a file.
 --
--- @since 0.7.0
-{-# INLINE appendChunks #-}
-appendChunks :: (MonadIO m, MonadCatch m)
+-- /Pre-release/
+--
+{-# INLINE writeAppendChunks #-}
+writeAppendChunks :: (MonadIO m, MonadCatch m)
     => FilePath -> Stream m (Array a) -> m ()
-appendChunks = fromChunksMode AppendMode
+writeAppendChunks = fromChunksMode AppendMode
```

Comment:
We can undo these renamings, in case anyone is using these they won't get affected. We anyway have to introduce writeAppend etc as Fold type.